src/SprykerEco/Zed/Payone/Business/Payment/Hook/PostSaveHookInterface.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\Hook;
+
+use Generated\Shared\Transfer\CheckoutResponseTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+
+interface PostSaveHookInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
```

Comment:
* The document block must look like this one and for other methods without any additional things.

Reply:
++